lib/utils.js

```diff
@@ -66,4 +69,22 @@ async function killAppUsingAppName (udid, appName) {
   }
 }
 
-export { detectUdid, getAndCheckXcodeVersion, getAndCheckIosSdkVersion, killAppUsingAppName };
+async function adjustWDAAttachmentsPermissions(perms) {
+  let processedFolders = [];
+  let derivedDataSearchMask = path.join(process.env['HOME'], WDA_DERIVED_DATA_SEARCH_SUFFIX);
```

Comment:
@mykola-mokhnach this still won't do the trick; if `process.env.HOME` is undefined, won't `path.join` throw an exception?

maybe an exception is what we want, but even so, we should provide a nice error message:

```js
if (!process.env.HOME) {
  throw new Error("Need HOME env var to be set in order to adjust WDA attachments permission");
}
```